CHANGELOG.md

```diff
@@ -9,6 +9,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [v1.7.0] - 2024-10-24
 
+### Changed
+- Update "sedge-data" default directory.
+
```

Comment:
Shouldn't this be under unreleased?

Reply:
Also, please share more details about the change, like what to expect from the new default data directory.

Reply:
Removed it from here since we will be adding these changes in a new PR. and updated changelog [here](https://github.com/NethermindEth/sedge/pull/464/commits/15d3b7d8e69370393c5e92aadd3ed5547874bc04).